### PR TITLE
set due in dashboard

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>
fixes: #1684

image

### CODE REVIEW

**Comment by raimund-schluessler** [comment]
@jakobroehrl Mind showing a screenshot of the Dashboard widget?

**Comment by jakobroehrl** [comment]
I get the error: "Missing required prop: "task""
What can I do?

File: src/components/TaskCreateDialog.vue [hunk 1]
```diff
@@ -45,6 +45,20 @@ License along with this library. If not, see <http://www.gnu.org/licenses/>.
 						:disabled="loading">
 				</div>
 
+				<DatetimePickerItem
+					:all-day="allDay"
+					:task:="newTask">
```
Comment (raimund-schluessler) [inline src/components/TaskCreateDialog.vue:50]:
```suggestion
					:task="newTask">
```

**Comment by raimund-schluessler** [comment]
> 
> 
> I get the error: "Missing required prop: "task""
> What can I do?

Well, provide the missing prop 🙈
The `DatetimePickerItem` needs the task prop, and I don't think an empty array will do. An empty object *might*  work, but I don't know for sure. Also, you have an extra colon in the code (see comment).

**Comment by jakobroehrl** [comment]
> > I get the error: "Missing required prop: "task""
> > What can I do?
> 
> Well, provide the missing prop 🙈
> The `DatetimePickerItem` needs the task prop, and I don't think an empty array will do. An empty object _might_ work, but I don't know for sure. Also, you have an extra colon in the code (see comment).

Thanks. Fixed it with: "newTask: {},"
But now I get:
[Vue warn]: Error in render: "TypeError: Cannot read properties of null (reading 'isValid')"

**Comment by jakobroehrl** [comment]
@raimund-schluessler 
What did you change? I can't see your changes, thanks!

**Comment by raimund-schluessler** [comment]
> 
> 
> @raimund-schluessler
> What did you change? I can't see your changes, thanks!

I didn't change anything yet, I just rebased to master.

**Comment by raimund-schluessler** [comment]
Closing for now, since I think when we find the time to implement #1684, a clean start will be better.

**Pull request closed without merging**